pkg/services/sqlstore/preferences.go

```diff
@@ -42,6 +42,7 @@ func (ss *SQLStore) GetPreferencesWithDefaults(ctx context.Context, query *model
 		}
 
 		res := &models.Preferences{
+			NavPosition:     ss.Cfg.NavPosition,
```

Comment:
All these backend changes need to be behind feature toggle, so we need to figure out how to pass the config.